auto_nag/scripts/inactive_ni_pending.py

```diff
@@ -0,0 +1,184 @@
+            self.add_action(bug)
+
+    def is_recent_bug(self, bug):
+        return lmdutils.get_date_ymd(bug["creation_time"]) >= self.__recent_bug_limit
+
+    def add_action(self, bug):
+        users_num = len(set([flag["requestee"] for flag in bug["inactive_ni"]]))
+
+        if (
+            (
+                bug["priority"] in HIGH_PRIORITY
+                or bug["severity"] in HIGH_SEVERITY
+                or self.is_recent_bug(bug)
+            )
+            and not utils.is_no_assignee(bug["triage_owner"])
+            and not any(
+                [
+                    flag["requestee"] == bug["triage_owner"]
+                    for flag in bug["inactive_ni"]
+                ]
+            )
+        ):
+            autofix = {
+                "flags": [
+                    {
+                        "id": flag["id"],
+                        "requestee": bug["triage_owner"],
+                    }
+                    for flag in bug["inactive_ni"]
+                ],
+                "comment": {
+                    "body": f"Redirect needinfo pending on inactive {utils.plural('user', users_num)} to the triage owner.",
+                },
+            }
+        else:
+            autofix = {
+                "flags": [
+                    {
+                        "id": flag["id"],
+                        "status": "X",
+                    }
+                    for flag in bug["inactive_ni"]
+                ],
+                "comment": {
+                    "body": f"Clear needinfo pending on inactive {utils.plural('user', users_num)}.",
+                },
+            }
+
+        self.add_prioritized_action(bug, bug["triage_owner"], autofix=autofix)
+
+    def get_bug_sort_key(self, bug):
+        return (
+            bug["priority"] not in HIGH_PRIORITY,
+            bug["severity"] not in HIGH_SEVERITY,
+            not self.is_recent_bug(bug),
+            utils.get_sort_by_bug_importance_key(bug),
+        )
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = {
+            "priority": bug["priority"],
+            "severity": bug["severity"],
+            "creation_time": bug["creation_time"],
+            "triage_owner": bug["triage_owner"],
+            "needinfo_flags": [
+                flag for flag in bug["flags"] if flag["name"] == "needinfo"
+            ],
+        }
+
+        return bug
+
+    def get_bz_params(self, date):
+        date = lmdutils.get_date_ymd(date)
+
+        fields = [
+            "triage_owner",
+            "flags",
+            "priority",
+            "severity",
+            "creation_time",
+        ]
+
+        params = {
+            "include_fields": fields,
+            "resolution": "---",
+            "f1": "flagtypes.name",
+            "o1": "equals",
+            "v1": "needinfo?",
+        }
+
+        # Run monthly on all bugs and weekly on recently created bugs
```

Comment:
Maybe recently changed is more interesting than recently created? How many results we get in dry run in weekly mode with the current parameters?

Reply:
I agree with you, recently changed is more interesting. However, we have a limitation with retrieving "recently changed" bugs. The query will include activities performed by bots or automation scripts. It is a solvable issue; we could filter the activities on the client side. But is it worth it? Anyway, we will include all bugs in the monthly run, so they will not be lifted behind.

In the weekly mode, we have 39 bugs.

Reply:
How many bugs would we have if we considered all recently changed, without filtering activity by bots?

> In the weekly mode, we have 39 bugs.

Wow, that's a lot! I thought we'd have very few bugs created in the last month with ni on inactive users! Could you share the full list so we can ensure they're all good results?

Reply:
> How many bugs would we have if we considered all recently changed, without filtering activity by bots?

In the weekly mode, we have 99 bugs.

> Wow, that's a lot! I thought we'd have very few bugs created in the last month with ni on inactive users! Could you share the full list so we can ensure they're all good results?

See the results below. It seems there is a limitation in the recent activity query; active users got classified as inactive.

## Dry-run in weekly mode - recently created

Returns 39 bugs, where all actions are needinfo forwarding.

<table style="border:1px solid black;border-collapse:collapse;" border="1">
<thead>
<tr>
<th>Bug</th><th>Summary</th><th>Requestee</th><th>Status</th>
</tr>
</thead>
<tbody>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1767081">1767081</a>
</td>
<td rowspan="1">
Windows 11, strange updating problems
</td>
<td>
tarapiccari@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1767009">1767009</a>
</td>
<td rowspan="1">
Right Click Context menu not working on items in bookmark menus
</td>
<td>
ablatt@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766773">1766773</a>
</td>
<td rowspan="1">
New tab in Firefox browser not working properly (new tab not opened)
</td>
<td>
maheshbitode1996@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766406">1766406</a>
</td>
<td rowspan="1">
Repaint issue with Browser Zoom and Auto-scroll to element in view
</td>
<td>
ganesh.zanjad@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766403">1766403</a>
</td>
<td rowspan="1">
Firefox crashes opening download list from download panel on Fedora 35.
</td>
<td>
ianpas78@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766319">1766319</a>
</td>
<td rowspan="1">
Draggable element not showing full content on first drag
</td>
<td>
robertsmrtn1@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766193">1766193</a>
</td>
<td rowspan="1">
Selecting text in browser window does not work for shift+right arrow.
</td>
<td>
mlshelton1975@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766144">1766144</a>
</td>
<td rowspan="1">
firefox unresponsive after waking from sleep
</td>
<td>
git@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766093">1766093</a>
</td>
<td rowspan="1">
Everything on youtube home page is highlighted blue/selected
</td>
<td>
matttewell@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766036">1766036</a>
</td>
<td rowspan="1">
Tab crashing and possible memory leak associated with playing video (YouTube, streams/Twitch)
</td>
<td>
gretchen.daehn@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765246">1765246</a>
</td>
<td rowspan="1">
firefox falsely expects SAIO and SAIZ boxes in AES-CTR protected audio streams in mp4
</td>
<td>
rjbean@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765129">1765129</a>
</td>
<td rowspan="1">
IDM extension not working properly
</td>
<td>
me.adanup@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765092">1765092</a>
</td>
<td rowspan="1">
When I enter on my blog from Mozilla, all style is crashed
</td>
<td>
xboxreviewer.com@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765057">1765057</a>
</td>
<td rowspan="1">
Firefox 99.0.1 snap firefox won&#39;t start with wayland + kernel driver nouveau (Failed to load module &#34;canberra-gtk-module&#34;)
</td>
<td>
vivien@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765056">1765056</a>
</td>
<td rowspan="1">
Firefox Welcome Destroys Session
</td>
<td>
ed@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764926">1764926</a>
</td>
<td rowspan="1">
Responsive Mode // Navigation within a specific website not posible
</td>
<td>
Rhett.Butler.108@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764805">1764805</a>
</td>
<td rowspan="1">
04-14-22 Auto Update to 99-0-1 causes the FireFox Browser to go &#34;non responsive&#34; changing Tabs
</td>
<td>
f_paz@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764691">1764691</a>
</td>
<td rowspan="1">
fxsnapshot analysis - view can not be set
</td>
<td>
stefan.zumbrunn+fire@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764643">1764643</a>
</td>
<td rowspan="1">
&#34;bookmarkbackups&#34; folder in my Firefox profile disappeared
</td>
<td>
tim@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764597">1764597</a>
</td>
<td rowspan="1">
mozilla update 99.0.1 not working on windows 7 , 64 bit after update
</td>
<td>
ayurvedicplants@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764558">1764558</a>
</td>
<td rowspan="1">
Firefox process use excessive amount of memory
</td>
<td>
destroyer954@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764533">1764533</a>
</td>
<td rowspan="1">
Changes panel is recording incorrect changes
</td>
<td>
saidalachgar.dev@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764528">1764528</a>
</td>
<td rowspan="1">
bookmark
</td>
<td>
monika123jaiswal@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764487">1764487</a>
</td>
<td rowspan="1">
Languages cannot be translated
</td>
<td>
sybca202024@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764253">1764253</a>
</td>
<td rowspan="1">
Paragraph spacing in Word online is huge and destroys layout
</td>
<td>
mail@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764062">1764062</a>
</td>
<td rowspan="1">
Layout broken with firefox 99.0 &amp; ubuntu 20.04
</td>
<td>
koopeepyy@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763981">1763981</a>
</td>
<td rowspan="1">
First full-screen video after launch badly positioned in Windows 11
</td>
<td>
ba001@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763978">1763978</a>
</td>
<td rowspan="1">
Widevine DRM breaks when using a non-default profile folder in 99.0 Windows-x64
</td>
<td>
anzz1@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763899">1763899</a>
</td>
<td rowspan="1">
Red and yellow XML error displayed (&amp;pictureInPictureToggle.label;) on videos
</td>
<td>
wch.ranksys@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763879">1763879</a>
</td>
<td rowspan="1">
Very high Firefox power usage (as per task manager, freezing up pc - major issues using youtube
</td>
<td>
xanangelofdeathx@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763870">1763870</a>
</td>
<td rowspan="1">
Firefox is already running, but is not responding. The old Firefox process must be closed to open a new window
</td>
<td>
troy-patterson@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763793">1763793</a>
</td>
<td rowspan="1">
can&#39;t load login screens for public wi-fi pages
</td>
<td>
jhchambliss@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763426">1763426</a>
</td>
<td rowspan="1">
crash after pulling a tab in fullmode
</td>
<td>
kajetann218@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763300">1763300</a>
</td>
<td rowspan="1">
A problem caused this program to stop interacting with Windows.
</td>
<td>
gnostatic@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763114">1763114</a>
</td>
<td rowspan="1">
Inconsistent tab order with Chrome
</td>
<td>
chy.bie@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762798">1762798</a>
</td>
<td rowspan="1">
Firefox glitches for less than a second when un-minimizing a window
</td>
<td>
firefoxuser932903@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762786">1762786</a>
</td>
<td rowspan="1">
javascript performance issues in active tab
</td>
<td>
bret.towe@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762756">1762756</a>
</td>
<td rowspan="1">
When pausing media, the audio stops in one AirPod before the other.
</td>
<td>
matthewrobertson513@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762739">1762739</a>
</td>
<td rowspan="1">
Spectrum internet pay site doesn&#39;t work.
</td>
<td>
al11588@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
</tbody>
</table>





## Dry-run in weekly mode - recently changed

- Total: 99 bugs 
- Actions: 
    - Forwarding: 70
    - Clearing: 29 bugs


<p>The following bugs have pending needinfo on inactive users:
<table style="border:1px solid black;border-collapse:collapse;" border="1">
<thead>
<tr>
<th>Bug</th><th>Summary</th><th>Requestee</th><th>Status</th>
</tr>
</thead>
<tbody>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1767081">1767081</a>
</td>
<td rowspan="1">
Windows 11, strange updating problems
</td>
<td>
tarapiccari@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1767009">1767009</a>
</td>
<td rowspan="1">
Right Click Context menu not working on items in bookmark menus
</td>
<td>
ablatt@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766773">1766773</a>
</td>
<td rowspan="1">
New tab in Firefox browser not working properly (new tab not opened)
</td>
<td>
maheshbitode1996@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766406">1766406</a>
</td>
<td rowspan="1">
Repaint issue with Browser Zoom and Auto-scroll to element in view
</td>
<td>
ganesh.zanjad@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766403">1766403</a>
</td>
<td rowspan="1">
Firefox crashes opening download list from download panel on Fedora 35.
</td>
<td>
ianpas78@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766319">1766319</a>
</td>
<td rowspan="1">
Draggable element not showing full content on first drag
</td>
<td>
robertsmrtn1@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766193">1766193</a>
</td>
<td rowspan="1">
Selecting text in browser window does not work for shift+right arrow.
</td>
<td>
mlshelton1975@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766144">1766144</a>
</td>
<td rowspan="1">
firefox unresponsive after waking from sleep
</td>
<td>
git@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766093">1766093</a>
</td>
<td rowspan="1">
Everything on youtube home page is highlighted blue/selected
</td>
<td>
matttewell@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1766036">1766036</a>
</td>
<td rowspan="1">
Tab crashing and possible memory leak associated with playing video (YouTube, streams/Twitch)
</td>
<td>
gretchen.daehn@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765246">1765246</a>
</td>
<td rowspan="1">
firefox falsely expects SAIO and SAIZ boxes in AES-CTR protected audio streams in mp4
</td>
<td>
rjbean@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765129">1765129</a>
</td>
<td rowspan="1">
IDM extension not working properly
</td>
<td>
me.adanup@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765092">1765092</a>
</td>
<td rowspan="1">
When I enter on my blog from Mozilla, all style is crashed
</td>
<td>
xboxreviewer.com@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765057">1765057</a>
</td>
<td rowspan="1">
Firefox 99.0.1 snap firefox won&#39;t start with wayland + kernel driver nouveau (Failed to load module &#34;canberra-gtk-module&#34;)
</td>
<td>
vivien@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1765056">1765056</a>
</td>
<td rowspan="1">
Firefox Welcome Destroys Session
</td>
<td>
ed@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764926">1764926</a>
</td>
<td rowspan="1">
Responsive Mode // Navigation within a specific website not posible
</td>
<td>
Rhett.Butler.108@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764805">1764805</a>
</td>
<td rowspan="1">
04-14-22 Auto Update to 99-0-1 causes the FireFox Browser to go &#34;non responsive&#34; changing Tabs
</td>
<td>
f_paz@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764691">1764691</a>
</td>
<td rowspan="1">
fxsnapshot analysis - view can not be set
</td>
<td>
stefan.zumbrunn+fire@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764643">1764643</a>
</td>
<td rowspan="1">
&#34;bookmarkbackups&#34; folder in my Firefox profile disappeared
</td>
<td>
tim@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764597">1764597</a>
</td>
<td rowspan="1">
mozilla update 99.0.1 not working on windows 7 , 64 bit after update
</td>
<td>
ayurvedicplants@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764558">1764558</a>
</td>
<td rowspan="1">
Firefox process use excessive amount of memory
</td>
<td>
destroyer954@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764533">1764533</a>
</td>
<td rowspan="1">
Changes panel is recording incorrect changes
</td>
<td>
saidalachgar.dev@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764528">1764528</a>
</td>
<td rowspan="1">
bookmark
</td>
<td>
monika123jaiswal@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764487">1764487</a>
</td>
<td rowspan="1">
Languages cannot be translated
</td>
<td>
sybca202024@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764253">1764253</a>
</td>
<td rowspan="1">
Paragraph spacing in Word online is huge and destroys layout
</td>
<td>
mail@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1764062">1764062</a>
</td>
<td rowspan="1">
Layout broken with firefox 99.0 &amp; ubuntu 20.04
</td>
<td>
koopeepyy@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763981">1763981</a>
</td>
<td rowspan="1">
First full-screen video after launch badly positioned in Windows 11
</td>
<td>
ba001@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763978">1763978</a>
</td>
<td rowspan="1">
Widevine DRM breaks when using a non-default profile folder in 99.0 Windows-x64
</td>
<td>
anzz1@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763899">1763899</a>
</td>
<td rowspan="1">
Red and yellow XML error displayed (&amp;pictureInPictureToggle.label;) on videos
</td>
<td>
wch.ranksys@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763879">1763879</a>
</td>
<td rowspan="1">
Very high Firefox power usage (as per task manager, freezing up pc - major issues using youtube
</td>
<td>
xanangelofdeathx@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763870">1763870</a>
</td>
<td rowspan="1">
Firefox is already running, but is not responding. The old Firefox process must be closed to open a new window
</td>
<td>
troy-patterson@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763793">1763793</a>
</td>
<td rowspan="1">
can&#39;t load login screens for public wi-fi pages
</td>
<td>
jhchambliss@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763426">1763426</a>
</td>
<td rowspan="1">
crash after pulling a tab in fullmode
</td>
<td>
kajetann218@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763300">1763300</a>
</td>
<td rowspan="1">
A problem caused this program to stop interacting with Windows.
</td>
<td>
gnostatic@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1763114">1763114</a>
</td>
<td rowspan="1">
Inconsistent tab order with Chrome
</td>
<td>
chy.bie@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762798">1762798</a>
</td>
<td rowspan="1">
Firefox glitches for less than a second when un-minimizing a window
</td>
<td>
firefoxuser932903@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762786">1762786</a>
</td>
<td rowspan="1">
javascript performance issues in active tab
</td>
<td>
bret.towe@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762756">1762756</a>
</td>
<td rowspan="1">
When pausing media, the audio stops in one AirPod before the other.
</td>
<td>
matthewrobertson513@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762739">1762739</a>
</td>
<td rowspan="1">
Spectrum internet pay site doesn&#39;t work.
</td>
<td>
al11588@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1762716">1762716</a>
</td>
<td rowspan="1">
Video freezing while screen sharing on discord
</td>
<td>
mikemizz006@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1761440">1761440</a>
</td>
<td rowspan="1">
Browser changes caret position to start of url
</td>
<td>
braechnov@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1761406">1761406</a>
</td>
<td rowspan="1">
Application unresponsiveness/parent process jank of 20-30s observed on Funimation show pages
</td>
<td>
brendan@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1760587">1760587</a>
</td>
<td rowspan="1">
Email bodies in Gmail are ignored by tab-key navigation
</td>
<td>
mail@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1760511">1760511</a>
</td>
<td rowspan="1">
Keyboard not recognized after opening a new tab.
</td>
<td>
oguz.uzman@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1759913">1759913</a>
</td>
<td rowspan="1">
Ver 98 with home opening multiple URLs when FF started
</td>
<td>
akilgore@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758659">1758659</a>
</td>
<td rowspan="1">
Intermittent TEST-UNEXPECTED-TIMEOUT | /fetch/private-network-access/service-worker-background-fetch.https.window.html | public to local: success. - Test timed out
</td>
<td>
wptsync@xxxx.xxx
</td>
<td>
Not specified
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758597">1758597</a>
</td>
<td rowspan="1">
Intermittent TEST-UNEXPECTED-TIMEOUT | /IndexedDB/idb-partitioned-basic.tentative.sub.html | Simple test for partitioned IndexedDB - Test timed out
</td>
<td>
wptsync@xxxx.xxx
</td>
<td>
Not specified
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758573">1758573</a>
</td>
<td rowspan="1">
Intermittent TEST-UNEXPECTED-TIMEOUT | /html/browsers/browsing-the-web/back-forward-cache/eligibility/dedicated-worker.html | Eligibility: dedicated workers - Test timed out
</td>
<td>
wptsync@xxxx.xxx
</td>
<td>
Not specified
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758305">1758305</a>
</td>
<td rowspan="1">
FireFox adds two records for insert into MySQL ASP
</td>
<td>
troymcc@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758300">1758300</a>
</td>
<td rowspan="1">
bug at startup : libc segfault / GFX1 : No GPUs detected via PCI
</td>
<td>
fxdeltombe@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758224">1758224</a>
</td>
<td rowspan="1">
I cannot input &#34;Chinese character&#34; into web_address column for searching purpose.
</td>
<td>
hunt_chang@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1758047">1758047</a>
</td>
<td rowspan="1">
Mozilla Firefox consumes high Background CPU and Memory Usage compared to other browsers
</td>
<td>
yashrudani.it@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1757387">1757387</a>
</td>
<td rowspan="1">
Cannot use a Data URL as a source for an &lt;audio&gt; tag for large files
</td>
<td>
florian.sw@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1757373">1757373</a>
</td>
<td rowspan="1">
Running ESLint across the entire tree (with ./mach eslint .) uses a single processor rather than multiple
</td>
<td>
nisargkarun1997@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1756855">1756855</a>
</td>
<td rowspan="1">
PDF Variable Fonts Display Incorrectly
</td>
<td>
jmanly3@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1755973">1755973</a>
</td>
<td rowspan="1">
Using WebGL2 returns wrong Uniform Buffer Object indices for uniform arrays
</td>
<td>
nikoloffgeorgi@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1755587">1755587</a>
</td>
<td rowspan="1">
Browser keeps getting stuck on certain sites
</td>
<td>
didilaws@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1754612">1754612</a>
</td>
<td rowspan="1">
Current time in Google Calendar is rendered incorrectly
</td>
<td>
dmalyshau@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1753590">1753590</a>
</td>
<td rowspan="1">
https://threadreaderapp.com does not work correctly with ETP set to STRICT
</td>
<td>
frank@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1753480">1753480</a>
</td>
<td rowspan="1">
Between V92 and 96 Missing Minimize/Window/Close on the Title Bar missing
</td>
<td>
billymays2000@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1752538">1752538</a>
</td>
<td rowspan="1">
AddressSanitizer: ILL on unknown address 0x7fea8fcf1c78 [@xxxx.xxx
</td>
<td>
dmalyshau@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1751191">1751191</a>
</td>
<td rowspan="1">
Some Chinese sites are broken in Firefox 96
</td>
<td>
yyzh888888@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1747608">1747608</a>
</td>
<td rowspan="1">
Window size &#39;clear recent history&#39;
</td>
<td>
gyo5h2pz76@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1745095">1745095</a>
</td>
<td rowspan="1">
Loading issues with Facebook
</td>
<td>
stefthibodeau@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1733936">1733936</a>
</td>
<td rowspan="1">
White empty page when other &#34;unrelated&#34; applications have the draw over other apps permission granted
</td>
<td>
contact@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1723357">1723357</a>
</td>
<td rowspan="1">
Form Resubmission Dialog is not shown
</td>
<td>
prashanith007@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1716541">1716541</a>
</td>
<td rowspan="1">
Implement Emulation.setTimezoneOverride
</td>
<td>
barnabas@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1707842">1707842</a>
</td>
<td rowspan="1">
Function of &#34;Match Diacritics&#34; button for Asian languages is not clearly explained
</td>
<td>
mdeboer@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1706692">1706692</a>
</td>
<td rowspan="1">
Third party cookies blocking of atoall.com by default is illegal
</td>
<td>
atoall11@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1691835">1691835</a>
</td>
<td rowspan="1">
Enhanced Tracking Protection with WPML wordpress plugin results in password fields staying as &#34;hidden&#34; on Windows 10 only
</td>
<td>
bkarabinakis@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1685882">1685882</a>
</td>
<td rowspan="1">
Wrong certificate issuer is displayed
</td>
<td>
fabian@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1670505">1670505</a>
</td>
<td rowspan="1">
RemoteSettings is stored in the disk cache in Firefox Focus
</td>
<td>
amy.dyne@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1667698">1667698</a>
</td>
<td rowspan="1">
Firefox does not send cookies for third party requests for js and css
</td>
<td>
lketired@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1667048">1667048</a>
</td>
<td rowspan="1">
FirefoxCP Web Content (several simultaneous processes) grows to several GB of memory, slows machine response.
</td>
<td>
ray.butler@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1665584">1665584</a>
</td>
<td rowspan="1">
Pip &#34;Mute&#34; button unmuted if &#34;Play&#34; button is clicked on Euronews.com and bbc.com
</td>
<td>
alexandra.martin@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1651434">1651434</a>
</td>
<td rowspan="1">
Tab loading takes a very long time after a new startup or a restart
</td>
<td>
andre.lemos893@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1610461">1610461</a>
</td>
<td rowspan="1">
Add tests for HyperTextAccessible::ScrollSubstringToPoint
</td>
<td>
apinheiro@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1544413">1544413</a>
</td>
<td rowspan="1">
Broken SVG with large coordinates (with Skia and WebRender backends, though not with Cairo)
</td>
<td>
arista@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1537925">1537925</a>
</td>
<td rowspan="1">
No visual hint that browser is no longer connected to sync after a password change or password reset
</td>
<td>
bbell@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1535466">1535466</a>
</td>
<td rowspan="1">
Should disable buffer-full-add-after-full-event web platform tests
</td>
<td>
whawkins@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1534120">1534120</a>
</td>
<td rowspan="1">
Crash in [@xxxx.xxx
</td>
<td>
mahmoud_salama8086@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1532441">1532441</a>
</td>
<td rowspan="1">
Update Deeplink Handler to Accept UTMs
</td>
<td>
fpatel@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1510811">1510811</a>
</td>
<td rowspan="1">
On-boarding prevents FxA deep-linking from working
</td>
<td>
fpatel@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1495307">1495307</a>
</td>
<td rowspan="1">
Setting CSS color on buttons leads to unreadable text with a dark desktop theme
</td>
<td>
punx665@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1479685">1479685</a>
</td>
<td rowspan="1">
www.taiwanbus.tw doesn&#39;t include the necessary intermediate certificates in its TLS handshake
</td>
<td>
hessi9@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1460412">1460412</a>
</td>
<td rowspan="1">
API to set the new tab page URL
</td>
<td>
tmijul1@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1434127">1434127</a>
</td>
<td rowspan="1">
Inorrect Japanese character in Yu Gothic font using standard variant sequence
</td>
<td>
lukasz.korczewski@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1423125">1423125</a>
</td>
<td rowspan="1">
Restore Previous Session Shortcut Key
</td>
<td>
mdeboer@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1412336">1412336</a>
</td>
<td rowspan="1">
Device orientation in A-Frame demo doesn&#39;t work
</td>
<td>
dmarcos@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1320745">1320745</a>
</td>
<td rowspan="1">
Download speeds limited on gigabit internet connection
</td>
<td>
mozilla@xxxx.xxx
</td>
<td>
Inactive on Bugzilla in last 26 weeks
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1265457">1265457</a>
</td>
<td rowspan="1">
Embedded Facebook videos blocked by tracking protection
</td>
<td>
jessjess1981@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1246236">1246236</a>
</td>
<td rowspan="1">
Implement local filesystem read/write access.
</td>
<td>
billjm05@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1235231">1235231</a>
</td>
<td rowspan="1">
Sessions restore windows in random order
</td>
<td>
mdeboer@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1147229">1147229</a>
</td>
<td rowspan="1">
Background images with negative background-position x are incorrectly printed
</td>
<td>
coq@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1103011">1103011</a>
</td>
<td rowspan="1">
Caret position inside empty editable element (input, textarea, contenteditable) with dir=auto doesn&#39;t respect parent&#39;s directionality
</td>
<td>
smontagu@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1092107">1092107</a>
</td>
<td rowspan="1">
Image is not updated on first reload
</td>
<td>
gamewithlalit@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=937494">937494</a>
</td>
<td rowspan="1">
Implement CSS matrix interpolation per the w3c spec
</td>
<td>
cabanier@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr >

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=922963">922963</a>
</td>
<td rowspan="1">
update Unicode Bidi Algorithm implementation to v6.3
</td>
<td>
ted.clancy@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
<tr bgcolor="#E0E0E0">

<td rowspan="1">
<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=907297">907297</a>
</td>
<td rowspan="1">
Improve resource usage HTML display
</td>
<td>
fjjdfsd@xxxx.xxx
</td>
<td>
Account disabled
</td>
</tr>
</tbody>
</table>

Reply:
Handled in c23c035cb66ca76beffb1c4b6c71cde21ce52919 and e28c653b413fc38fcabf66bf9a632b9247794efa